.github/workflows/main.yml

```diff
@@ -7,12 +7,8 @@ jobs:
     strategy:
       matrix:
         env:
-          - {ROS_DISTRO: dashing,  ROS_REPO: testing}
-          - {ROS_DISTRO: dashing,  ROS_REPO: main}
-          - {ROS_DISTRO: eloquent, ROS_REPO: testing}
-          - {ROS_DISTRO: eloquent, ROS_REPO: main}
-          - {ROS_DISTRO: foxy,     ROS_REPO: testing}
-          - {ROS_DISTRO: foxy,     ROS_REPO: main}
+          - {ROS_DISTRO: galactic,  ROS_REPO: testing}
```

Comment:
I'm not sure we want to remove foxy yet, because that's still a supported ROS distro.

Reply:
I'd agree, but some of the changes break compatibility with foxy. Specifically, rclcpp::Duration::from_nanoseconds() doesn't exist in foxy (according to the api documentation at least).